Backend-tests.md

```diff
@@ -214,18 +216,23 @@ python -m scripts.run_backend_tests --generate_coverage_report
 If there are **any** backend test errors, no coverage report will be produced. Please fix those errors and then re-run the above command. If the tests all pass, a coverage report will be printed that lists each backend file, along with the lines not covered by tests. Here is an example of a coverage report:
 
 ```text
```

Comment:
This does get printed in the console, but since there are no console commands here, I don't think the console syntax highlighting makes sense here (it just turns all the text blue because there's no `$`)